ethanschreur/kids-krafts
# Sellers can add orders

**What does this PR do?**
- This pull request allows sellers to add customer orders

**Description of the task to be completed** 
- Add a form  on the "orders" page that can be used to add orders to the database

**How should this  be tested**
- (On Windows) Run `. venv/scripts/activate` 
- `python run.py`
- Go to http://127.0.0.1:5000/orders (log in if you have to)
- Fill out the "Add Orders" form and enter it
- Check that the order you added is now displayed in the "Orders" table
